AutoClicker/Views/MainWindow.xaml.cs

```diff
@@ -183,10 +184,33 @@ private void AboutCommand_Execute(object sender, ExecutedRoutedEventArgs e)
 
         private int CalculateInterval()
         {
-            return AutoClickerSettings.Milliseconds
-                + (AutoClickerSettings.Seconds * 1000)
-                + (AutoClickerSettings.Minutes * 60 * 1000)
-                + (AutoClickerSettings.Hours * 60 * 60 * 1000);
+            var clickInterval = AutoClickerSettings.Milliseconds
+                                + (AutoClickerSettings.Seconds * 1000)
+                                + (AutoClickerSettings.Minutes * 60 * 1000)
+                                + (AutoClickerSettings.Hours * 60 * 60 * 1000);
+
+            // if the clickInterval is 0, check the randomizedClickInterval
+            if (clickInterval == 0)

```

Comment:
It could be nice to have
```
if (clickInterval == 0) {
    return calculateRandomInterval()
}
return calculateFixedInterval()
```

Reply:
I was considering adding a separate flag to enable/disable random intervals. I think the extra clarity that an independant flag adds is convient. As I mentioned in a couple of the commit messages, I still need to preform quite a few cleanup tasks. This will probably be one of them.

While not quite the same as your suggestion, I think we are aiming for the same goal of making it obvious to future users when the randomize interval is used. I personally prefer a flag for this purpose since it is the first thing I would think to look for

Reply:
After a reasonable amount of effort, I have decided it isn't worthwhile to figure out how to implement the flag. Apparently it is not as simple as I had hoped. Instead I've implemented your suggestion.